recipes/jasper/all/conanfile.py

```diff
@@ -92,11 +95,43 @@ def package(self):
             for dll_file in glob.glob(os.path.join(self.package_folder, "bin", "*.dll")):
                 if os.path.basename(dll_file).startswith(("concrt", "msvcp", "vcruntime")):
                     os.unlink(dll_file)
+        self._create_cmake_module_variables(
+            os.path.join(self.package_folder, self._module_file_rel_path)
+        )
+
+    @staticmethod
+    def _create_cmake_module_variables(module_file):
+        content = textwrap.dedent("""\
+            if(DEFINED Jasper_FOUND)
```

Comment:
seems to be one more repeating pattern - mapping CMake variables, besides one for creating alias targets

Reply:
In the conan issues for this, it was not desired to add this ... perhaps proposing a new tool would be possible?

Reply:
yes, new tool using build modules